### ISSUE TITLE
Allow using SQLite?

### ISSUE THREAD

**Issue by stubailo** [abhiaiyer91/meteor-pg-apollo-shopping-cart#3]
Since you're using Sequelize, it should be easy to use SQLite as well, which might save people from needing to set up Postgres?

**Comment by abhiaiyer91** [comment]
Sure! I just thought PG would be flashy.

**Comment by stubailo** [comment]
Absolutely! I just think it would simplify the setup instructions so people could get going faster. If you don't think it's important feel free to close the issue :]

**Comment by abhiaiyer91** [comment]
Nah, very important! I'll make an Options object people can configure!
